tests/baselines/reference/tsserver/autoImportProvider/Auto-importable-file-is-in-inferred-project-until-imported.js

```diff
@@ -89,12 +89,8 @@ TI:: [hh:mm:ss:mss] Got install request
       },
       "unresolvedImports": [],
       "projectRootPath": "/node_modules/@angular/forms",
-      "cachePath": "/a/data",
```

Comment:
Removes unnecessary addition of cachePath that was done by test typing installer. (its same as globalcache path that will be processed at the initialization and was not set by node typings installer as well)